vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/promotion/delete_claim/delete_claim.cc

```diff
@@ -0,0 +1,118 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/endpoint/promotion/delete_claim/delete_claim.h"
+
+#include <utility>
+
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/common/request_util.h"
+#include "bat/ledger/internal/endpoint/promotion/promotions_util.h"
+#include "bat/ledger/internal/ledger_impl.h"
+#include "net/http/http_status_code.h"
+
+using std::placeholders::_1;
+
+namespace {
+
+std::string GetPath(const std::string custodian,
+                    const std::string& payment_id) {
+  return base::StringPrintf("/v3/wallet/%s/%s/claim", custodian.c_str(),
+                            payment_id.c_str());
+}
+
+}  // namespace
+
+namespace ledger {
+namespace endpoint {
+namespace promotion {
+
+DeleteClaim::DeleteClaim(LedgerImpl* ledger) : ledger_(ledger) {
+  DCHECK(ledger_);
+}
+
+DeleteClaim::~DeleteClaim() = default;
+
+std::string DeleteClaim::GetUrl(const std::string custodian) {
```

Comment:
I think this should be `const std::string&`.

Reply:
resolved